src/views/Sandbox/index.tsx

```diff
@@ -1,16 +1,75 @@
 import React, { useEffect } from 'react';
+import { Button } from '@trussworks/react-uswds';
 
 import Header from 'components/Header';
+import SystemProfileHealthCard from 'components/SystemProfileHealthCard';
 
 const Sandbox = () => {
   useEffect(() => {
     document.title = 'Sandbox';
   }, []);
+
   return (
     <div>
```

Comment:
Rather than
```
<div>
  <Header />
  (grid-container div)
</div>
```
the usual wrapper is
```
<PageWrapper>
  <Header />
  <MainContent className="gridContainer">
    (div with grid-col-10)
  </MainContent>
  <Footer />
</PageWrapper>
```

Reply:
Addressed in 6efe49bc